ShinyExpenseTracker/src/ca/ualberta/cs/shinyexpensetracker/models/ExpenseItem.java

```diff
@@ -133,4 +134,12 @@ public void setReceiptPhoto(Bitmap photo) {
 	public Bitmap getReceiptPhoto(){
 		return this.receiptPhoto;
 	}
+
+	public boolean doesHavePhoto() {
+		return hasPhoto;
```

Comment:
Why doesn't this check if `this.receiptPhoto` is null or not instead of using its own value? If you have a good reason to do it this way, you should add a comment explaining that.